addons/jest/src/components/Panel.tsx

```diff
@@ -149,28 +158,94 @@ const Content = styled(({ tests, className }: ContentProps) => (
                 >
                   <div
                     id="failing-tests"
-                    title={`${testsByType.get(StatusTypes.FAILED_TYPE).length} Failed`}
+                    title={`${
+                      testsByType.get(StatusTypes.FAILED_TYPE)
+                        ? testsByType.get(StatusTypes.FAILED_TYPE).length
+                        : 0
+                    } Failed`}
                     color={getColorByType(StatusTypes.FAILED_TYPE)}
                   >
                     <List>
-                      {testsByType.get(StatusTypes.FAILED_TYPE).map((res: any) => (
-                        <Item key={res.fullName || res.title}>
-                          <Result {...res} />
-                        </Item>
-                      ))}
+                      {testsByType.get(StatusTypes.FAILED_TYPE) ? (
```

Comment:
@CodeByAlex lots of duplicate code here. make a component?

Reply:
Agreed, it should be